src/assets/translations/de-DE.json

```
@@ -2661,7 +2661,7 @@
       "Provisioning Criteria": "Bereitstellungskriterien",
       "Purchase Price Ratio": "Kaufpreisverhältnis",
       "Purchased/Redeemed Price": "Gekaufter/eingelöster Preis",
```

Biome (2.4.15)

[error] The key Purchased/Redeemed Price was already declared.

(lint/suspicious/noDuplicateObjectKeys)